src/Store.js

```diff
@@ -13,14 +14,18 @@ if (process.env.NODE_ENV !== 'production') {
   middlewares.push(logger)
 }
 
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
I'm not familiar with this tool... is it worth introducing to the codebase?

Reply:
I actually didn't think twice about this since it doesn't actually add the dependency, it just exposes to the window. I'm happy to remove it and I can just add it locally if I need to debug.